shap/maskers/_text.py

```diff
@@ -122,6 +136,9 @@ def token_segments(self, s):
     def clustering(self, s):
         self._update_s_cache(s)
         decoded_x = [self.tokenizer.decode([v]) for v in self._tokenized_s]
+        # TODO: temp fix for sente piece tokenizer
+        if self.keep_suffix>0 and len(decoded_x[-1])==0:
```

Comment:
seems like we need linting in shap? as I would assume you would want to have spaces between the comparisons, eg self.keep_suffix > 0.  Hmm I thought we did run flake8, maybe I'm mistaken.

Reply:
yeah, it just got added in a recent PR but will take more work to get repo-wide